react-1/src/components/User/User.js

```diff
@@ -0,0 +1,77 @@
+import React, { useImperativeHandle, useRef } from "react";
+import { useStyles } from "./style";
+import { DragSource, DropTarget } from "react-dnd";
+import ItemTypes from "./ItemTypes";
+
+const User = React.forwardRef((props, ref) => {
```

Comment:
With hooks API (`useDrag/useDrop`) you wouldn't need to have the ref of your element. See http://react-dnd.github.io/react-dnd/examples/sortable/simple